rules/libyang3.mk

```diff
@@ -0,0 +1,28 @@
+# libyang3
+
+LIBYANG3_VERSION = 3.7.8
+LIBYANG3_SUBVERSION = 3
+LIBYANG3_FULLVERSION = $(LIBYANG3_VERSION)-$(LIBYANG3_SUBVERSION)
+
+export LIBYANG3_VERSION
+export LIBYANG3_SUBVERSION
+export LIBYANG3_FULLVERSION
+
+LIBYANG3 = libyang3_$(LIBYANG3_FULLVERSION)_$(CONFIGURED_ARCH).deb
+$(LIBYANG3)_SRC_PATH = $(SRC_PATH)/libyang3
+SONIC_MAKE_DEBS += $(LIBYANG3)
+
+LIBYANG3_DEV = libyang-dev_$(LIBYANG3_FULLVERSION)_$(CONFIGURED_ARCH).deb
```

Comment:
Please rename to `libyang3-dev_...` to stick to existing naming convention

Reply:
You're reviewing part of a PR that's squashed into this one.  Please review each commit independently and ignore the commits that say to ignore.  Anyhow, that original PR has been merged.